src/seismometer/data/pandas_helpers.py

```diff
@@ -76,6 +88,12 @@ def merge_windowed_event(
     ValueError
         At least one column in pks must be in both the predictions and events dataframes.
     """
+    # Validate merge strategy
+    if merge_strategy not in get_args(MergeStrategies):
```

Comment:
Not an issue, I just wanted to say this is a slick way to get the list.

Reply:
[Credit goes to](https://github.com/epic-open-source/seismometer/pull/76#discussion_r1731142510) @diehlbw :)